osdk/src/config/test/OSDK.toml.full

```diff
@@ -0,0 +1,67 @@
+project_type = "kernel"
+
+supported_archs = ["x86_64"]
+vars = [
+    ["SMP", "1"],
+    ["MEM", "2G"],
+    ["EXT2_IMG", "$OSDK_CWD/regression/build/ext2.img"],
+    ["EXFAT_IMG", "$OSDK_CWD/regression/build/exfat.img"],
+]
+
+[boot]
+method = "grub-rescue-iso"
+
+[run]
+vars = [
+    ["OVMF_PATH", "/usr/share/OVMF"],
+]
+boot.kcmd_args = [
+    "SHELL=/bin/sh",
+    "LOGNAME=root",
+    "HOME=/",
+    "USER=root",
+    "PATH=/bin:/benchmark",
+    "init=/usr/bin/busybox",
+]
+boot.init_args = ["sh", "-l"]
+boot.initramfs = "/tmp/osdk_test_file"
+
+[test]
+boot.method = "qemu-direct"
+
+[grub]
+protocol = "multiboot2"
+skip_menu = true
+debug_print = false
+
+[qemu]
+args = """\
+    -machine q35 \
+    -smp $SMP \
+    -m $MEM \
+"""
+
+[scheme."iommu"]
+supported_archs = ["x86_64"]
+vars = [
+    ["IOMMU_DEV_EXTRA", ",iommu_platform=on,ats=on"],
+]
+qemu.args = """\
+    -device intel-iommu,intremap=on,device-iotlb=on \
+    -device ioh3420,id=pcie.0,chassis=1\
+"""
+
+[scheme."tdx"]
```

Comment:
The `tdx` scheme is only a placeholder, or is used to show some OSDK features that are not used in other schemas?

If former is true, I suggest removing the schema context since it only contains random values.

Reply:
Um, but it tests the feature selection parsing, which is exactly needed by TDX.

I don't have much motivation making it random now.